### ISSUE TITLE
3 CCHF sequences belong together

### ISSUE THREAD

**Issue by corneliusroemer** [pathoplexus/curation_reports#4]
These 3 sequences belong into 1 Pathoplexus sequence, the reason they weren't grouped is because the isolate name has the segment as suffix which makes our heuristic force them to be in different sequences.

- https://pathoplexus.org/seq/PP_000SMLM.1
- https://pathoplexus.org/seq/PP_000SMNH.1
- https://pathoplexus.org/seq/PP_000SMMK.1

<img width="1793" alt="Google Chrome Beta 2024-11-17 23 42 06" src="https://github.com/user-attachments/assets/32f0cd6f-eb9b-4d1c-9901-10ed72d763de">

<img width="1777" alt="Google Chrome Beta 2024-11-17 23 42 03" src="https://github.com/user-attachments/assets/7bc4cbce-2ab7-4658-9ff6-9ada91b11854">

<img width="1828" alt="Google Chrome Beta 2024-11-17 23 41 56" src="https://github.com/user-attachments/assets/c081da0a-655f-4fc2-b2d9-0decbc2cd8d2">

I think this can't be done at curation level right now, we might need some sort of grouping override in ingest.

**Comment by arthurshem86** [comment]
I agree with you. These should be grouped under the same Pathoplexus ID.  These are different segments of the same isolate. Even the authors and submission dates on the gene bank records are the same. <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ463982.1">segment L</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ463984.1">segment S</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ463983.1">segment M</a>
```
 AUTHORS: Peng,L., Hirsch,S., Kocher,G., Mehta,M. and Holbrook,M.
 JOURNAL: Submitted (10-OCT-2024) IRF, NIH/NIAID, 8200 Research Plaza,
            Frederick, MD 21702, USA
  ```

**Comment by anna-parker** [comment]
Hi @arthurshem86 thank you so much for looking into and confirming this regrouping! Sadly as we discussed in our morning dev meeting there is no easy way for us to curate segment regroupings of CCHF sequences via the webpage. 

Currently we do this by adding the curated groups into a JSON file and this is used as an "overwrite" when we ingest sequences from the INSDC: https://github.com/pathoplexus/curation_data/blob/main/grouping_override/cchf/curated_group_13-02-2026.json. We think it would be best for us developers to add this grouping overwrite there and then we will revoke the existing single sequences with a comment linking to this issue. Thank you for your help - I will let you know when the changes are on the webpage!

**Comment by arthurshem86** [comment]
Sure!. Thanks a lot.

**Comment by anna-parker** [comment]
this has now been resolved :-)
